lib/model/migrations/20231002-01-add-conflict-details.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2023 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = async (db) => {
+  await db.raw('ALTER TABLE entities ADD COLUMN conflict VARCHAR(10) NULL');
```

Comment:
`varchar` because we should store type of conflict like:

- `NULL`: no conflict
- `soft`: version discrepancy
- `hard`: two properties modified simultaneously 

maybe in future we need to show type of conflict on home/entity-list page

Reply:
What do you think about using an `ENUM`?